build.yaml

```diff
@@ -783,6 +783,8 @@ steps:
       script: /io/sql/insert_globals.py
     - name: create-billing-projects
       script: /io/sql/create-billing-projects.sql
+    - name: migration-1
+      script: /io/sql/migration-1.sql
```

Comment:
Can we give these short, readable names, like a git branch name?  `add-user-resource-token`, maybe?